script.js

```diff
@@ -6,4 +9,317 @@ function openNav(){
+        if (typeof(currCheckedSeason) !== "undefined")
+        {
+            document.cookie = "currCheckedSeason="+currCheckedSeason+
+            "; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/";
+        }
+    };
+
+    function getCurrCheckedSeason($param) {
+        var currCheckedSeason = $param.val();
+        if (typeof(currCheckedSeason) !== "undefined")
+        {
+            return currCheckedSeason;
+        }
+    }
+
+    //if hits enter during season text box, trigger button
+    $('#newSeasonInputBox').keypress(function(e){
+        if (e.keyCode == 13)
+        {
+            addNewSeason();
+        }
+    });
+
+    //if hits enter while entering points, trigger save button
+    $('#elementPoints').keypress(function(e){
+        if (e.keyCode == 13)
+        {
+            $('#scoringSaveButton').click();
+        }
+    });
+
+    //adds new element to table, handles things like element already exists and season not checked
+    $("#scoringSaveButton").click(function(){
+        var newElementName = $("#elementKey").val().trim().toString();
+        var newElementPoints = $("#elementPoints").val().trim();
+        
+        if (newElementName !== "" || newElementPoints !== "")
+        {
+            var season = $("input[name=seasonListItem]:checked","#seasonRadioList").val();
+            if (typeof(season) !== "undefined")
+            {
+                var cookieName = '/season_config/' + season + '/' + newElementName;
+                var cookieValue = newElementPoints;
+                
+                var cookieList = document.cookie.split(";");
+                var elementAlreadyStored = false;
+                for (var i = 0; i < cookieList.length;i++)
+                {
+                    if (cookieList[i].trim().startsWith('/season_config/' + season))
+                    {
+                        var str = cookieList[i];
+                        var checkerName = str.substring(str.lastIndexOf('/')+1,str.indexOf('='));
+                        if (checkerName == newElementName)
+                        {
+                            elementAlreadyStored = true;
+                            break;
+                        }
+                    }
+                }
+                
+                if (!elementAlreadyStored)
+                {
+                    $("#elementConfirmationBox").text(" ");
+                    document.cookie = cookieName + "=" + cookieValue + 
+                    "; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/";
+                    //doc.cookie --> /season_config/season/elementName=elementValue;...
+
+                    var newTRHtml = "<tr><td>" + newElementName + 
+                    "</td><td>" + newElementPoints + "</td></tr>";
+                    
+                    $("#scoringTable").append(newTRHtml);
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" added.<\i>');
+
+                    $("#elementKey").val("");
```

Comment:
for ease of use, tweak this line to be:
```JS
$("#elementKey").val("").focus();
```

This will move the cursor back to the key field

This allows for:
```
Ball
TAB
5
Enter
Hatch
10
```

quicker input if I don't have to shift+tab back (or click)
You'll see this as well with Excel - start entering a row of items, enter 1, tab, enter a second, tab, enter a third, enter - it will take you to the next row at the column you entered the first item in - allow for quickly typing out a lot of data.

Reply:
That's a cool feature, thanks.